src/db/arby-order.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  DataTypes,
+  Model,
+  ModelAttributes,
+  ModelOptions,
+  Sequelize,
+} from 'sequelize';
+import BigNumber from 'bignumber.js';
+import { ModelCtor } from 'sequelize/types/lib/model';
+
+enum ArbyOrderSide {
+  BUY,
+  SELL,
+}
+
+type ArbyOrderCreationAttributes = {
+  cexOrderId: string;
+  openDexOrderId: string;
+  side: ArbyOrderSide;
+  quantity: BigNumber;
+  price: BigNumber;
+  time: number;
+  profit: BigNumber;
+};
+
+type ArbyOrderAttributes = ArbyOrderCreationAttributes;
+
+export interface ArbyOrderInstance
+  extends Model<ArbyOrderAttributes, ArbyOrderCreationAttributes>,
+    ArbyOrderAttributes {}
+
+export function ArbyOrder(sequelize: Sequelize): ModelCtor<ArbyOrderInstance> {
+  const attributes: ModelAttributes<ArbyOrderInstance> = {
+    cexOrderId: { type: DataTypes.STRING, primaryKey: true },
+    openDexOrderId: { type: DataTypes.STRING, allowNull: false },
+    side: { type: DataTypes.TINYINT, allowNull: false },
```

Comment:
The order side is always both as arbitrage trade's are always asymmetrical.

Reply:
It was in the issue, so I guess this's the side of CEX order as far as I understood.